Sources/Prometheus/MetricTypes/Summary.swift

```diff
@@ -84,9 +67,7 @@ public class PromSummary<NumType: DoubleRepresentable, Labels: SummaryLabels>: P
         self.capacity = capacity
 
         self.quantiles = quantiles
-        
-        self.labels = labels
```

Comment:
Parent Summary was always created with empty labels. subHistogram labels were stored twice: as a key in `subSummaries` and as a field in corresponding `subSummary` value. I just removed the duplication.